src/ingestion/sources/redshift.py

```diff
@@ -12,6 +12,12 @@ def _connection_string_prefix(self) -> str:
     def type(self):
         return "redshift"
 
+class RedshiftSourceDialect(PostgreSQLSourceDialect):
+    @classmethod
+    def _freshness(cls):
+        return "DATEDIFF(MINUTE, MAX(CAST({} AS TIMESTAMP)), GETDATE())"
```

Comment:
Knowing that current_timestamp is available on redshift, GETDATE worries me that we lose fidelity considering this is in minutes.

Reply:
Used `GETDATE()` because Redshift docs stated that `CURRENT_TIMESTAMP` was deprecated 

https://docs.aws.amazon.com/redshift/latest/dg/Date_functions_header.html